content-cache/src/nginx_manager.py

```diff
@@ -217,7 +248,7 @@ def _create_status_page_config() -> None:
             )
         )
     )
-    _create_and_enable_config("nginx_status", nginx_config)
+    _store_and_enable_site_config("nginx_status", nginx_config)
 
 
 def _create_server_config(
```

Comment:
As "Server" is an nginx keyword refering to a different section than the one we are generating here. I suggest to be more explicit by renaming the function create_virtualhost_config (or create_site_config). It will make the scope clearer.

Reply:
Updated.